docusaurus/docs/guides/kubernetes/workload-tunneling/index.md

```diff
@@ -38,34 +39,38 @@ In the chart above, there are several strategies and solutions for connecting a
 
 An OpenZiti Tunneler can be used to intercept pod egress. One important thing to know is that the OpenZiti Tunneler that is used in this way may also be used to "host" OpenZiti Services in order to provide ingress to cluster services.
 
-#### [Node Proxy Daemonset](./kubernetes-daemonset.md)
+<!-- #### [Node Proxy Daemonset](./kubernetes-daemonset.md)
 
-Deploying a daemonset of privileged `ziti-edge-tunnel run` pods on selected nodes is a simple way to enable OpenZiti services in a cluster. The daemonset pods intercept egress from pods and provide a DNS nameserver for CoreDNS. Like any other OpenZiti Tunneler, the OpenZiti Identity used by the daemonset may be configured to host OpenZiti Services, i.e. provide OpenZiti ingress to cluster services. For more information, see the [node proxy](./kubernetes-daemonset.md) page.
+Deploying a daemonset of privileged `ziti-edge-tunnel run` pods on selected nodes is a simple way to enable OpenZiti services in a cluster. The daemonset pods intercept egress from pods and provide a DNS nameserver for CoreDNS. Like any other OpenZiti Tunneler, the OpenZiti Identity used by the daemonset may be configured to host OpenZiti Services, i.e. provide OpenZiti ingress to cluster services. For more information, see the [node proxy](./kubernetes-daemonset.md) page. -->
 
-#### [Transparent Proxy Sidecar](./kubernetes-sidecar.md)
+#### [ziti-tunnel as Transparent Proxy Sidecar](./kubernetes-sidecar.md)
```

Comment:
imo, I'd rather not highlight ziti-tunnel in this way. not to mention it's not "ziti-tunnel" anymore...

Reply:
The name of the container image is `docker.io/openziti/ziti-tunnel`. It doesn't make sense to obscure the usefulness of the `ziti-tunnel` image because there are no alternatives for either of the use cases listed in this article:

1. localhost tcp proxy
2. transparent proxy

Reply:
@dovholuknf Here is a table showing the things that only `ziti tunnel` can do: https://github.com/openziti/ziti-doc/issues/383